unified_targets/configs/MANGOF405.config

```diff
@@ -0,0 +1,126 @@
+# Betaflight / MANGOF405 (S405) 4.0.3 Jun 26 2019 / 10:17:27 () MSP API: 1.41
+
+board_name MANGOF405
+manufacturer_id CUST
```

Comment:
If this is not an official target it will not be added to the repository. Like I said in #8465, please read and follow the instruction, or find somebody who is capable of doing so.

Also, you need to stop abusing GitHub by closing pull requests and opening them again with essentially the same content. This is wasting everybody's time and will get you banned.